gradle/static-analysis.gradle

```diff
@@ -29,7 +29,7 @@ apply plugin: 'pmd'
  * See https://github.com/gradle/gradle/issues/8277.
  */
 pmd {
-    toolVersion = "${deps.versions.pmd}"
+    toolVersion = '6.16.0'
```

Comment:
Why don't we use that version any more?

Reply:
The `deps` property is no longer exposed by the `bootstrap` plugin, starting from around version `0.15.7`. 

There seems to be nothing in its place to acquire the dependency versions from.